src/app/components/Figure/Caption/index.stories.jsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import VisuallyHiddenText from '../../VisuallyHiddenText';
 import Caption from './index';
 
 storiesOf('Caption', module).add('default', () => (
```

Comment:
I'd suggest having two stories here. One which is without the visually hidden text and one without it. This will highlight the different ways of using the component.

Reply:
I'd disagree with this as the `VisuallyHiddenText` is an accessibility requirement so making it look optional in the storybook examples is misleading for people who reuse this component in the future. Due to the fact that Figure is actually what passes in the visually hidden text it's currently difficult to ensure the offscreen text is passed into the caption. However in the future when translations are being passed into component we would be able to ensure an `offscreenCaptionText` prop is passed in and then passed to the Caption component. Currently we will have to hardcode this

Reply:
@pjlee11 If you want the visually hidden text to be an immovable part of the Caption, you should not include it here in the Figure in this way. It should be included in the Caption, in here (https://github.com/bbc/simorgh/blob/latest/src/app/components/Figure/Caption/index.jsx).  

Then within `Figure`, you would make no changes at all. 

Thinking through this point, I'd recommend taking this approach, since the `VisuallyHiddenText` shouldn't be passed in as a child element of the Figure.